config/routes.rb

```diff
@@ -121,50 +121,45 @@
 
     get "/scanned_resources/:parent_id/new", to: "scanned_resources#new", as: :parent_new_scanned_resource
 
-    resources :numismatic_references
-    get "/numismatic_references/:parent_id/new", to: "numismatic_references#new", as: :parent_new_numismatic_reference
-
-    resources :numismatic_accessions
-
-    resources :numismatic_finds
-
-    resources :numismatic_firms
-
-    resources :numismatic_places
-
-    resources :numismatic_people
-
-    resources :numismatic_issues do
-      member do
-        get :order_manager
-        get :manifest, defaults: { format: :json }
-        post :browse_everything_files
+    namespace :numismatics do
+      resources :accessions
+      resources :coins do
+        member do
+          get :file_manager
+          get :order_manager
+          get :manifest, defaults: { format: :json }
+          get :orangelight, defaults: { format: :json }
+          post :browse_everything_files
+          get :discover_files
+          post :auto_ingest
+          get :pdf
+        end
       end
-    end
-
-    resources :numismatic_monograms do
-      member do
-        get :file_manager
-        get :order_manager
-        get :manifest, defaults: { format: :json }
-        post :browse_everything_files
+      resources :issues do
+        member do
+          get :order_manager
+          get :manifest, defaults: { format: :json }
+          post :browse_everything_files
+        end
       end
-    end
-
-    resources :coins do
-      member do
-        get :file_manager
-        get :order_manager
-        get :manifest, defaults: { format: :json }
-        get :orangelight, defaults: { format: :json }
-        post :browse_everything_files
-        get :discover_files
-        post :auto_ingest
-        get :pdf
+      resources :finds
+      resources :firms
+      resources :people
+      resources :places
+      resources :monograms do
+        member do
+          get :file_manager
+          get :order_manager
+          get :manifest, defaults: { format: :json }
+          post :browse_everything_files
+        end
       end
+      resources :references
     end
-    get "/numismatic_issues/:parent_id/coin" => "coins#new", as: :parent_new_coin
-    get "/parent/:parent_id/numismatic_artist" => "numismatic_artists#new", as: :parent_add_numismatic_artist
```

Comment:
I'm not seeing Numismatics::Artist in the routes — is this not needed (only used nested)?

Reply:
Artist is nested. I think line 167 was leftover from an earlier refactor.